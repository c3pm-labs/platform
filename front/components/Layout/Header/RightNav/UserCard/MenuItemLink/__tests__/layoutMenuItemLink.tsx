import { render } from '@testing-library/react';

import MenuItemLink from '../index';

test('Layout MenuItemLink', () => {
  const { container } = render(
    <MenuItemLink
      className=""
      // eslint-disable-next-line no-console
      onClick={() => console.log('close')}
      href="/user/[params]"
      as="/user/10"
    >
      toto
    </MenuItemLink>,
  );

  expect(container).toMatchSnapshot();
});
