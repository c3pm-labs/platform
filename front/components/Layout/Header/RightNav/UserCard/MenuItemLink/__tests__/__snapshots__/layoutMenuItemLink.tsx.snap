// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout MenuItemLink 1`] = `
<div>
  <a
    aria-disabled="false"
    class="MuiButtonBase-root MuiListItem-root MuiMenuItem-root MuiMenuItem-gutters MuiListItem-gutters MuiListItem-button"
    href="/user/10"
    role="menuitem"
    tabindex="-1"
  >
    toto
    <span
      class="MuiTouchRipple-root"
    />
  </a>
</div>
`;
